lib/add-allegation-data.js

```diff
@@ -6,9 +6,9 @@ const valuesMap = require('./ims-hof-values-map.json');
 
 const addFieldToEform = (eform, fieldName, fieldValue) => {
   if (fieldName === 'images') {
-    eform.Attachments = fieldValue;
-  } else if (fieldName === 'txpermoreallholder') {
-    eform.AdditionalPeople.push({FieldName: fieldName, FieldValue: fieldValue});
+    eform.Attachements = fieldValue;
```

Comment:
Make `Attachements` into `Attachments` here (typo but the conditional for attachments needs 'Attachments')